platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/sources/GeoJsonSource.java

```diff
@@ -133,8 +132,7 @@ public GeoJsonSource(String id, FeatureCollection features, GeoJsonOptions optio
    * @param features the GeoJSON {@link FeatureCollection}
    */
   public void setGeoJson(FeatureCollection features) {
```

Comment:
Not necessarily, no. This is a left-over from a method which had some additional checking on the jvm side before passing it on to the native method.